### PR TITLE
Merge preview8 into main

### PR DESCRIPTION
**Pull request by roji**
Merge preview8 into main after #21785

### CODE REVIEW

File: src/EFCore.Relational/Properties/RelationalStrings.Designer.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (smitpatel) [inline src/EFCore.Relational/Properties/RelationalStrings.Designer.cs:1]: extra empty line.

Comment (roji) [reply src/EFCore.Relational/Properties/RelationalStrings.Designer.cs:1]: Good catch. Submitted #21789 to fix this in release/5.0-preview8 first (please approve), after that's merged I'll redo this PR.

Comment (smitpatel) [reply src/EFCore.Relational/Properties/RelationalStrings.Designer.cs:1]: Not a functional impact. Just make it part of merge commit and it is fine.

Comment (roji) [reply src/EFCore.Relational/Properties/RelationalStrings.Designer.cs]: Done

**Comment by smitpatel** [comment]
Already been merged.

**Pull request closed without merging**